src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -180,6 +209,260 @@ In the example above, the files required are placed within the `/tmp` directory.
+  bind {{haproxyHost}}:5601
+  mode http
+  acl kibana_context_root path_beg /
+  use_backend kibana_backend if kibana_context_root
+
+backend kibana_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+  {{#ifAcl 'cinnamon-grafana-docker' '2' 'grafana'}}
+# Cinnamon - Grafana HAProxy Configuration
+frontend grafana_frontend
+  bind {{haproxyHost}}:3000
+  mode http
+  acl grafana_context_root path_beg /
+  use_backend grafana_backend if grafana_context_root
+
+backend grafana_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+  {{#ifAcl 'visualizer' '2' 'visualizer'}}
+# ConductR - Visualizer Bundle HAProxy Configuration
+frontend visualizer_frontend
+  bind {{haproxyHost}}:9999
+  mode http
+  acl visualizer_context_root path_beg /
+  use_backend visualizer_backend if visualizer_context_root
+
+backend visualizer_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+{{/eachAcls}}
+
+
+{{#haproxyConf bundles}}
+{{serviceFrontends}}
+{{#unless (serviceFrontends)}}
+frontend dummy
+  bind 127.0.0.1:65535
+{{/unless}}
+{{serviceBackends}}
+{{/haproxyConf}}
+```
+
+Here are the explanation for the example template above:
+
+* *HAProxy Specific Configuration* section is declared prior *Custom Endpoint Configuration* section.
+* *Custom Endpoint Configuration* section contains application specific proxy configuration.
+* The HTTP endpoint is exposed on the port `9000` by default.
+* The backend server information will be updated whenever there's changes to the endpoint, e.g. bundle instances being started or stopped the within the ConductR cluster.
+
+The custom configuration is declared within the [ifAcl](#ifAcl) block:
+
+Bundle Name             | System Version | Endpoint Name | HTTP Port
+------------------------|----------------|---------------|----------
+conductr-elasticsearch  |              1 | es            | 9200
+conductr-kibana         |              1 | kibana        | 5601
+conductr-grafana-docker |              2 | grafana       | 3000
+visualizer              |              2 | visualizer    | 9999
+
+These HTTP ports by will be exposed to the HAProxy docker container when the Sandbox starts.
+
+### Package the custom HAProxy configuration template
+
+Use the CLI to package the configuration override:
+
+```lang-none
+shazar /tmp/custom-haproxy-conf
+Created digested ZIP archive at ./custom-haproxy-conf-ffd0dcf76f4d565424a873022fbb39f3025d4239c87d307be3078b320988b052.zip
+
+```
+
+The generated file `custom-haproxy-conf-ffd0dcf76f4d565424a873022fbb39f3025d4239c87d307be3078b320988b052.zip` is the configuration override that can be loaded alongside ConductR HAProxy bundle.
+
+
+### Load the custom HAProxy configuration template
```

Comment:
You need to unload the default conductr-haproxy from the sandbox as well, otherwise you will have two running?